system/logaccess/ElasticStack/ElasticStackLogAccess.cpp

```diff
@@ -425,10 +425,23 @@ void esMatchQueryString(std::string & search, const char *searchval, const char
     search += "\" }";
 }
 
+const char * sortByDirectionToES(SortByDirection direction)
```

Comment:
minor: I see the change, the function is now a static member of the class, does it need to be a member? (is it going to be shared with anything else including the header?

In fact, I don't think ElasticStackLogAccess should be exported (ELASTICSTACKLOGACCESS_API) at all?
Since it is implementation only, never intended to be shared with anything else I think, only the factory method wants exposing and that's all (the interface, IRemoteLogAccess, is already exposed).

Normally, we would declare an implementation only class like this in the cpp, or implement the methods inline with the declaration of the class in the cpp.
If the implementation class is shared with other implementations in other files, then it's worth putting it in it's own implementation header.

Reply:
@jakesmith the comment about not exporting the implementations is reasonable and will be handled separately:
https://track.hpccsystems.com/browse/HPCC-27772